.gitignore

```diff
@@ -0,0 +1,34 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules
+jspm_packages
+
+# Optional npm cache directory
+.npm
+
+# Optional REPL history
+.node_repl_history
```

Comment:
There's nothing wrong with it. I just want to be sure that you understand what it's doing. I though that it may have been overwritten by another plugin installation. It seems OK but there is a lot of unnecessary code in there as well e.g. excluding '.npm'. Copy and paste is ok for small things but you got to be careful when you do that for files that could affect the behaviour of the entire repo.